tests/Transfer/E2E/Sources/SourceCore.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Utopia\Tests\E2E\Sources;
+
+use PHPUnit\Framework\TestCase;
+use Utopia\Tests\E2E\Adapters\MockDestination;
+use Utopia\Transfer\Destination;
+use Utopia\Transfer\Resource;
+use Utopia\Transfer\Source;
+use Utopia\Transfer\Transfer;
+
+abstract class SourceCore extends TestCase
```

Comment:
```suggestion
abstract class Base extends TestCase
```
Maybe base is a better name ?